grails-app/controllers/org/pih/warehouse/picklist/PicklistImportDataCommand.groovy

```diff
@@ -0,0 +1,29 @@
+package org.pih.warehouse.picklist
+
+import org.pih.warehouse.api.PickPageItem
+import org.pih.warehouse.api.StockMovement
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.importer.ImportDataCommand
+
+class PicklistImportDataCommand extends ImportDataCommand {
+
+    StockMovement stockMovement
+    List<PickPageItem> pickPageItems
+    List<PicklistItemCommand> picklistItems
+
+    PicklistImportDataCommand() {
+        // FIXME this probably shouldn't be necessary, but i was getting
+        //  a validation error on location (possibly when the command object
+        //  was initially bound).
+        location = AuthService.currentLocation
```

Comment:
I think it should be a StockMovement's origin location instead current location. What do you think?